### ISSUE TITLE
For .headerImg, does file type matter? and other edge cases

### ISSUE THREAD

**Issue by kaymckelly** [kaymckelly/freelance-iacr#64]
Can it be animated? What happens if it's too small? If "too large" is even a thing, does it affect anything other than load time? Etc

**Comment by kaymckelly** [comment]
**Testing edge cases**

Note: all testing done locally in current stable release of chrome (56.0.2924.87)

tl;dr version: use 1200x480 images saved in .jpg, .png, or .gif formats

- Shorter than expected: larger space from bottom of gradient to text but gradient mostly fills it in; proceed with caution
- Narrower than expected: appears to size correctly, may cause stretching or other distortion with some images, proceed with caution
- Shorter and narrower than expected: appears to size correctly, may cause stretching or other distortion with some images, proceed with caution
- Animated: ok but may be extremely annoying, proceed with extreme caution
- .gif file extension: ok
- .png file extension: ok
- .bmp file extension: could not find testing file, assume incompatible unless proven otherwise
- .jpg file extension: ok
- .svg file extension: ok (may not render in all browsers, particularly <=IE8)
- Other file extension: not compatible, use a standard file format for your images!
